commands/push-image.ts

```diff
@@ -1,14 +1,29 @@
 import vscode = require('vscode');
 import { ImageItem, quickPickImage } from './utils/quick-pick-image';
 import { reporter } from '../telemetry/telemetry';
+import { DockerNode } from '../explorer/dockerExplorer';
 const teleCmdId: string = 'vscode-docker.image.push';
 
-export async function pushImage() {
+export async function pushImage(context?: DockerNode) {
+    let imageToPush: Docker.ImageDesc;
```

Comment:
Is this variable even used? Could we simply track the image name and use it to determine whether we push below?